geolocator_platform_interface/lib/src/enums/location_accuracy_status.dart

```diff
@@ -0,0 +1,8 @@
+/// Represent the current Location Accuracy Status on iOS devices
```

Comment:
nit: Also here it would be good to mention the version of iOS:

```suggestion
/// Represent the current Location Accuracy Status on iOS 14.0 and higher.
///
/// The [LocationAccuracyStatus] type is not used on pre-iOS 14.0 or other
/// platforms.
```